guides/git-pr-review.md

```diff
@@ -0,0 +1,53 @@
+# Github PR Review ("Code Review")
```

Comment:
Is it "Github" or "GitHub"? I feel like we need to be consistent on this.
```suggestion
# GitHub PR Review ("Code Review")
```

Reply:
fixed